extensions/micrometer-registry-prometheus/runtime/pom.xml

```diff
@@ -32,7 +32,13 @@
 
         <dependency>
             <groupId>io.micrometer</groupId>
-            <artifactId>micrometer-registry-prometheus-simpleclient</artifactId>
+            <artifactId>micrometer-registry-prometheus</artifactId>
+        </dependency>
+
+        <!--Excluded in micrometer but needed for native. Includes OpenTelemetry interaction utilities.-->
```

Comment:
Why is this needed for native?

Reply:
It's a bit weird to me as well. 

Micrometer seems not to want to rely on OpenTelemetry maybe because they use Brave for tracing.
To create exemplars they have this bit of code: 
https://github.com/prometheus/client_java/blob/b1ff4701a5c268f742dfc440308ea888b88e3ceb/prometheus-metrics-core/src/main/java/io/prometheus/metrics/core/exemplars/ExemplarSampler.java#L336

Because we use our own span context coming from OTel, we need to that part that is excluded by default by MM.

Reply:
Usually the solution to this kind of thing is to have a GraalVM substitution instead of adding dependencies we don't explicitly need.

Reply:
Fixed. A substitution did it.